xcode/Subconscious/Shared/Components/Settings/GatewayURLSettingsView.swift

```diff
@@ -52,7 +34,10 @@ struct GatewayProvisionLabel: View {
                             .repeatForever(autoreverses: false)
                             .speed(0.4), value: spin)
                         .onAppear() {
-                            self.spin = true
+                            // Avoids unwanted animation of position after layout
+                            DispatchQueue.main.asyncAfter(deadline: .now()) {
+                                self.spin = true
+                            }
```

Comment:
This is stupid but it seems like the right fix based on my googling. Without it I get this:



https://github.com/subconsciousnetwork/subconscious/assets/5009316/d8a30d14-1cb2-47a4-bd9d-47a6d98fba78